### DISCUSSION TITLE
Save log of Spam-comments, when ran out of quota

### DISCUSSION THREAD

**Discussion by JM-Lemmi** [ThioJoe/YT-Spammer-Purge#391] [Ideas]
When running out of quota while scanning, you should be able to save a log file to not have to repeat the scanning the next day. Instead all the already scanned comments can be instantly reported, as soon as more API quota is availible.

**Comment by ThioJoe** [comment]
Yea I'm definitely planning on adding additional features related to 'continuation' I guess you could call it. The addition of the mode that lets you delete using a list of comments is just the first. 

I'm probably first going to add a way to partially delete a comment list, then continue. Because at least when beginning scanning comments, you know how many you're going to need to scan, so you can have a better idea of whether you'll go over the limit (the daily limit default is 1 Million comments scanned, if nothing else is done). Whereas you won't know how many spam comments you'll find, and the effective daily limit there is 10,000.

Anyway but yea I'm definitely going to add features like you suggest.

**Reply by OfficialEsco** [reply]
Heeey, so, do you have a status update for this feature?
Scanning MrBeast videos is a honeypot of spam (9000) and getting limited is roooough after checking the false-positives (200-300 users)...

Not sure how deleting from log works, but i hope it marks them as deleted/reported so we don't go over them twice?
